dev-requirements.txt

```diff
@@ -6,4 +6,4 @@ pytest==8.3.3
 robotframework==7.2
 types-PyYAML
 types-requests
-
+diffoscope
```

Comment:
While this may be nice for development and testing, does it really need to be used for comparison, where you would expect the images to always be the same, i.e. just use diff